test-dssspell.sh

```diff
@@ -9,7 +9,7 @@ export DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0x9a6c490
 export DAPP_LINK_TEST_LIBRARIES=0
 
 if [[ -z "$1" ]]; then
-  dapp --use solc:0.6.11 test --rpc-url="$ETH_RPC_URL" -v
+  dapp --use solc:0.6.12 test --rpc-url="$ETH_RPC_URL" -v
```

Comment:
Remove the `=` after `--rpc-url`. Not sure about this, but at least on my machine it fails to test otherwise.

Reply:
Make sure you are on latest dapptools. They've changed a lot recently.